fdbclient/NativeAPI.actor.cpp

```diff
@@ -2562,6 +2581,13 @@ ACTOR static Future<Void> tryCommit( Database cx, Reference<TransactionLogInfo>
 	if (info.debugID.present())
 		TraceEvent(interval.begin()).detail( "Parent", info.debugID.get() );
 
+	if(CLIENT_BUGGIFY) {
+		throw g_random->randomChoice(std::vector<Error>{
+				not_committed(),
+				transaction_too_old(),
+				commit_unknown_result()});
```

Comment:
We could probably add `proxy_memory_limit_exceeded` to this list.

Reply:
good idea - added that error

Reply:
I don't see this error in the list.

Reply:
missed that one. It is now there.